update profiling page (#6715)
getsentry/sentry-docs: src/docs/product/profiling/index.mdx

</Note>

> Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring] (/product/performance/) capabilities to provide precise code-level visibility into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.

Author: ```suggestion
Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring](/product/performance/) capabilities to provide precise code-level visibility into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.
```

Sentry profiling supports common platforms for both Mobile and Backend applications:

> - [Mobile app profiling](/product/profiling/getting-started/mobile-app-profiling) makes it possible to collect **profiles from real devices in production**, providing a complete picture of how the app performs in a variety of environments.

Author: ```suggestion
- [Mobile app profiling] NEED LINK makes it possible to collect **profiles from real devices in production**, providing a complete picture of how the app performs in a variety of environments.
```

- Backend platform profiling makes it possible to easily target the essential code that yields the biggest improvements in backend throughput and resource utilization.

## Zooming In From Performance To Profiling

Sentry profiling data is structured as an extension of performance transactions.  From the Sentry **Performance** page, you can review high level transaction data from your application to identify suspect spans - operations that may be impacting performance, including slow DB queries, HTTP requests, and so on.  You can click on a span to see profiling information, including the most frequently occurring code path (call stack with with exact line numbers), along with the approximate percentage of time required for that code path.

> [Image 1]
> 
> From this view, you can click “View Profile” to zoom in on the flame chart. (How to read a flame chart link)
> 
> [Image 2]
> 
> ## Profiling Page
> 
> You can go to the Profiling page to see a list of transactions in descending order of execution time for your selected project(s) and also view the three “Slowest Transactions” broken down further with a list of the slowest identified functions. The “Profiles by Percentiles” timeline chart can be used to filter the transaction data by highlighting the desired time period.  Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.
> 
> [Image 3]
> 
> To zoom in further, select from the list of “Slowest App Functions” and see the associated flame chart for the profile.  (*How to read a flame chart link*)
> 
> Selecting one of the transactions will take you to the “Transaction Profile” summary page below.  On this page, you can examine the consolidated flame graph (*How to read flame graph link*) that presents aggregated sample data for the selected transaction.
> 
> [Image 4]

Author: ```suggestion

![profiling-information-span](https://user-images.githubusercontent.com/26800704/234110886-7b3912d1-883f-4227-b40e-62c84965c460.png)

From this view, you can click “View Profile” to zoom in on the flame chart. (How to read a flame chart link)

![profiling-summary-view](https://user-images.githubusercontent.com/26800704/234110914-f99519ba-e469-43e0-9579-6aeee7c3186b.png)

## Profiling Page

You can go to the Profiling page to see a list of transactions in descending order of execution time for your selected project(s) and also view the three “Slowest Transactions” broken down further with a list of the slowest identified functions. The “Profiles by Percentiles” timeline chart can be used to filter the transaction data by highlighting the desired time period.  Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.

![profiling-page-view](https://user-images.githubusercontent.com/26800704/234110952-aca7822e-d55c-4f69-8fc2-8a8989c800ac.png)

To zoom in further, select from the list of “Slowest App Functions” and see the associated flame chart for the profile.  (*How to read a flame chart link*)

Selecting one of the transactions will take you to the “Transaction Profile” summary page below.  On this page, you can examine the consolidated flame graph (*How to read flame graph link*) that presents aggregated sample data for the selected transaction.

![transaction-profile-view](https://user-images.githubusercontent.com/26800704/234111012-56e3e3dd-69e1-4925-9b25-4e1477223aad.png)

```

</Note>

> Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring](/product/performance/) capabilities to provide precise, code-level insight into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.

Reviewer: > manual test instrumentation

This sounds weird to me. What does manual test instrumentation mean here? @JonasBa can you confirm this is the best way to talk about this?

> Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring](/product/performance/) capabilities to provide precise code-level visibility into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.

Author: ```suggestion
Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring](/product/performance/) capabilities to provide precise, code-level insight into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.
```

Sentry profiling supports common platforms for both Mobile and Backend applications:

> - [Mobile app profiling] NEED LINK makes it possible to collect **profiles from real devices in production**, providing a complete picture of how the app performs in a variety of environments.

Author: ```suggestion
- [Mobile app profiling](/product/profiling/mobile-app-profiling/) makes it possible to collect **profiles from real devices in production**, providing a complete picture of how the app performs in a variety of environments.
```

- Backend platform profiling makes it possible to easily target the essential code that yields the biggest improvements in backend throughput and resource utilization.

## Zooming In From Performance To Profiling

Sentry profiling data is structured as an extension of performance transactions.  From the Sentry **Performance** page, you can review high level transaction data from your application to identify suspect spans - operations that may be impacting performance, including slow DB queries, HTTP requests, and so on.  You can click on a span to see profiling information, including the most frequently occurring code path (call stack with with exact line numbers), along with the approximate percentage of time required for that code path.

![profiling-information-span](https://user-images.githubusercontent.com/26800704/234110886-7b3912d1-883f-4227-b40e-62c84965c460.png)

> From this view, you can click “View Profile” to zoom in on the flame chart. (How to read a flame chart link)

Author: ```suggestion
From this view, you can click “View Profile” to zoom in on the flame chart. 
```

![profiling-summary-view](https://user-images.githubusercontent.com/26800704/234110914-f99519ba-e469-43e0-9579-6aeee7c3186b.png)

## Profiling Page

> You can go to the Profiling page to see a list of transactions in descending order of execution time for your selected project(s) and also view the three “Slowest Transactions” broken down further with a list of the slowest identified functions. The “Profiles by Percentiles” timeline chart can be used to filter the transaction data by highlighting the desired time period.  Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.
> 

Author: ```suggestion
You can go to the **Profiling** page to see a list of transactions in descending order of execution time for your selected project(s) and also view the three “Slowest Transactions” broken down further with a list of the slowest identified functions. The “Profiles by Percentiles” timeline chart can be used to filter the transaction data by highlighting the desired time period.  Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.

```

> ![profiling-page-view](https://user-images.githubusercontent.com/26800704/234110952-aca7822e-d55c-4f69-8fc2-8a8989c800ac.png)

Reviewer: ```suggestion
![profiling-page-view](profiling-page-view.png)
```

> To zoom in further, select from the list of “Slowest App Functions” and see the associated flame chart for the profile.  (*How to read a flame chart link*)

Author: ```suggestion
To zoom in further, select from the list of “Slowest App Functions” and see the associated flame chart for the profile.  
```

> Selecting one of the transactions will take you to the “Transaction Profile” summary page below.  On this page, you can examine the consolidated flame graph (*How to read flame graph link*) that presents aggregated sample data for the selected transaction.

Author: ```suggestion
Selecting one of the transactions will take you to the “Transaction Profile” summary page below.  On this page, you can examine the consolidated flame graph that presents aggregated sample data for the selected transaction.
```
